user_docs/en/userGuide.t2t

```diff
@@ -764,6 +764,10 @@ Although a physical mouse or trackpad should be used to navigate with the mouse,
 | Left mouse button lock | shift+numpadDivide | NVDA+control+[ | none | Locks the left mouse button down. Press again to release it. To drag the mouse, press this key to lock the left button down and then move the mouse either physically or use one of the other mouse routing commands |
 | Right mouse click | numpadMultiply | NVDA+] | tap and hold | Clicks the right mouse button once, mostly used to open context menu at the location of the mouse. |
 | Right mouse button lock | shift+numpadMultiply | NVDA+control+] | none | Locks the right mouse button down. Press again to release it. To drag the mouse, press this key to lock the right button down and then move the mouse either physically or use one of the other mouse routing commands |
+| Mouse wheel scroll up | none | none | none | Scrolls up the mouse wheel at the current mouse position |
+| Mouse wheel scroll down | none | none | none | Scrolls down the mouse wheel at the current mouse position |
+| Mouse wheel scroll left | none | none | none | Scrolls left the mouse wheel at the current mouse position |
+| Mouse wheel scroll right | none | none | none | Scrolls right the mouse wheel at the current mouse position |

```

Comment:
Unassigned shortcuts are usually not listed in the user doc's tables and in key commands. There is #13024 though about it. And there are two exceptions in the User Guide (Move review cursor to status bar and Report current symbol replacement).

NV Access (@seanbudd, @Qchristensen) could you clarify what is expected? Here and in #13024? Thanks.

Reply:
This has been answered in https://github.com/nvaccess/nvda/issues/13024#issuecomment-2103673460 now

I think these should remain documented